app/src/lib/pocket.ts

```diff
@@ -0,0 +1,56 @@
+import { Configuration, HttpRpcProvider, Pocket, PocketRpcProvider, QueryAppResponse, RpcError, typeGuard } from "@pokt-network/pocket-js"
+import { getAddressFromPublicKey } from '../utils/conversions';
```

Comment:
Haha, this has a semicolon and the other lines don't. We def need to set up some linting :)